src/poseidon/septidon/control.rs

```diff
@@ -1,7 +1,8 @@
 use super::params::GATE_DEGREE_5;
 use super::util::query;
+use halo2_proofs::arithmetic::FieldExt;
 use halo2_proofs::circuit::{Region, Value};
-use halo2_proofs::halo2curves::bn256::Fr as F;
+//use halo2_proofs::halo2curves::bn256::Fr as F;
```

Comment:
gm :) , just reading through scroll repos and trying to familiarize myself with the codebase. I was wondering why comment out an import like this rather than just remove it, if it's not used can't you just remove and it re-import it later if it is needed? I've noticed lots of commented out code-blocks throughout the scroll repos and was just curious if there was any reason they aren't just removed, or if the code is needed just make an issue to correct the code so that can be fixed and uncommented? It can be confusing to readers like me when wondering if the code is still relevant or not

Reply:
I think it is just a tip about "there had been an import" and the colleagues may get some hints from that. Anyway this should be simply a personal favourite / coding fashion.